src/sdk/SourceLanguageConfig.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { Contracts } from "./Contracts";
+
+export class SourceLanguageConfig {
```

Comment:
SDK classes need comments for the auto-ref docs.

Reply:
added